ballerina/client.bal

```diff
@@ -52,7 +52,8 @@ public type Client client object {
     # + return - Metadata of the query execution as an `sql:ExecutionResult[]` or an `sql:Error`
     remote isolated function batchExecute(ParameterizedQuery[] sqlQueries) returns ExecutionResult[]|Error;
 
-    # Executes an SQL query, which calls a stored procedure. This may or may not return results.
+    # Executes an SQL query, which calls a stored procedure. This may or may not
+    # return results. Once the results are processed, invoke the `close` method on the `sql:ProcedureCallResult`
```

Comment:
```suggestion
    # Executes an SQL query, which calls a stored procedure. This may or may not
    # return results. Once the results are processed, invoke the `close` method on the `sql:ProcedureCallResult`.
```